cmd/sonobuoy/app/images.go

```diff
@@ -101,3 +153,205 @@ func getImages(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 }
+
+func pullImages(cmd *cobra.Command, args []string) {
+	switch imagesflags.plugin {
+	case "e2e":
+
+		cfg, err := imagesflags.kubeconfig.Get()
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "couldn't get REST client"))
+			os.Exit(1)
+		}
+
+		sbc, err := getSonobuoyClient(cfg)
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "could not create sonobuoy client"))
+			os.Exit(1)
+		}
+
+		version, err := sbc.Version()
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "couldn't get Sonobuoy client"))
+			os.Exit(1)
+		}
+
+		upstreamImages, err := image.GetImages("", version)
```

Comment:
nit: Probably should const the empty string here so it has better meaning in the code. As a reader of the code I'd understand it is some default value, but I dont really know what. Maybe just a name like `const defaultE2ERegistries = ""`?